duckduckgo/content-scope-scripts
# ntp: update notification + context menu fixes

**Asana Task/Github Issue:** 

## Description

- [x] added support for an 'updateNotification'
- [x] moved more docs from JS files into markdown

## Testing Steps

- see https://deploy-preview-1178--content-scope-scripts.netlify.app/build/pages/new-tab/?update-notification=empty
- and https://deploy-preview-1178--content-scope-scripts.netlify.app/build/pages/new-tab/?update-notification=populated

## Checklist


*Please tick all that apply:*
